tools/check_symbols.py

```diff
@@ -21,7 +21,7 @@ def __str__(self):
 
   @staticmethod
   def parse(line):
-    return Symbol(line[:8].strip(), line[9], line[11:].strip())
+    return Symbol(line[:8].strip(), line[9], line[11:].strip().split('@')[0])
```

Comment:
Could you add a comment line with an example of the line to parse?

Reply:
Actually there's no real good example but I'll try.